nens_auth_client/views.py

```diff
@@ -25,7 +25,7 @@
 )
 
 
-def _get_redirect_from_next(request, default):
+def _get_redirect_from_next(request, default=None):
```

Comment:
Op zich kan de "default" ook weg, want het is nu niet echt meer een default.
Hij wordt verderop ook gebruikt met `_get_redirect() or settings.DEFAULT_DINGUS`.
Dus dan hoeft 'ie hier alleen het ding uit de next param terug te geven als dat er is. Niet gevonden? Dan heb je automatisch de impliciete `return None`

Default kan beter op die andere plek (waar je 'em ook al zo gebruikt).